src/lib/modules/tests/test.js

```diff
@@ -262,6 +269,13 @@ class Test {
     return instance;
   }
 
+  track(jsonInterface, address) {
```

Comment:
Is this function used?

Reply:
Yes it is used by user doing test and adding contract manually:
https://github.com/alaibe/liquidpledging/blob/feat/code-coverage/test/helpers/deployLP.js#L57

Any better idea?

Reply:
Nah, it's fine. I just didn't see any reference to it.